scripts/deployToForkedChain.js

```diff
@@ -0,0 +1,138 @@
+/* global hre */
+const stream = require("stream");
+const path = require("path");
+const { promisify } = require("util");
+
+const { ethers } = require("hardhat");
+const fs = require("fs");
+const axios = require("axios");
+
+const provider = new ethers.providers.StaticJsonRpcProvider("http://127.0.0.1:8545");
+
+const { FORKED_NETWORK_ADDRESS, REPUTATION_URL } = process.env;
+
+if (!FORKED_NETWORK_ADDRESS) {
+  throw new Error("FORKED_NETWORK_ADDRESS must be set");
+}
+
+if (!REPUTATION_URL) {
+  throw new Error("REPUTATION_URL must be set");
+}
+
+let signer;
+async function getResolverAddress(etherRouterAddress) {
+  const er = await ethers.getContractAt("EtherRouter", etherRouterAddress);
+  return er.resolver();
+}
+
+async function setResolver(etherRouterAddress, resolverAddress) {
+  const er = await ethers.getContractAt("EtherRouter", etherRouterAddress, signer);
+  return er.setResolver(resolverAddress);
+}
+
+async function copyResolver(fromAddress, toAddress) {
+  const fromResolver = await getResolverAddress(fromAddress);
+  return setResolver(toAddress, fromResolver);
+}
+
+async function getMostRecentEvent(contract, _filter) {
+  const filter = _filter;
+  const latestBlockNumber = await provider.getBlockNumber();
+  filter.fromBlock = latestBlockNumber + 1; // +1 to accommodate the first loop iteration
+  filter.toBlock = filter.fromBlock;
+  let foundEvent = false;
+  let events;
+  const BLOCK_PAGING_SIZE = 1000;
+  while (filter.toBlock > 0 && !foundEvent) {
+    // Create a span of events up to BLOCK_PAGING_SIZE in length
+    filter.toBlock = filter.fromBlock - 1;
+    filter.fromBlock = Math.max(filter.toBlock - BLOCK_PAGING_SIZE + 1, 0);
+
+    events = await provider.getLogs(filter);
+    if (events.length > 0) {
+      foundEvent = true;
+    }
+  }
+  return events[0];
+}
+
+async function main() {
+  signer = await ethers.getImpersonatedSigner("0x56a9212f7f495fadce1f27967bef5158199b36c7");
+  const forkedBlock = await provider.getBlockNumber();
+
+  await hre.run("deploy");
+
+  const DEPLOYED_NETWORK_ADDRESS = "0x777760996135F0791E2e1a74aFAa060711197777";
```

Comment:
Ditto this -- where is this coming from?

Reply:
It's the address `ColonyNetwork` is deployed to in production.